File: Android /Android/Separate Android /Android/JMBA_Olo_Android_SDK/app/src/main/java/com/fishbowl/apps/olo/Models/OloBillingAccount.java
package com.fishbowl.apps.olo.Models;

/**
 * Created by Ihsanulhaq on 5/29/2015.
 */
public class OloBillingAccount {

    private int accountid;
    private String accounttype;
    private String cardtype;
    private String cardsuffix;
    private String description;
    private String expiration;

    public int getAccountid() {
        return accountid;
    }

    public String getAccounttype() {
        return accounttype;
    }

    public String getCardtype() {
        return cardtype;
    }

    public String getCardsuffix() {
        return cardsuffix;
    }

    public String getDescription() {
        return description;
    }

    public String getExpiration() {
        return expiration;
    }
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.1/JMBA_Olo_iOS_SDK/Olo/OloBillingAccount.swift
//
//  OloBillingAccount.swift
//  Pods
//
//  Created by Taha Samad on 6/23/15.
//
//

import Foundation
import SwiftyJSON

public struct OloBillingAccount {
    
    public var accountId: Int64
    public var accountType: String
    public var cardType: String
    public var cardSuffix: String
    public var desc: String
    public var expiration: String
    
    public init(json: JSON) {
        accountId   = json["accountid"].int64Value
        accountType = json["accounttype"].stringValue
        cardType    = json["cardtype"].stringValue
        cardSuffix  = json["cardsuffix"].stringValue
        desc        = json["description"].stringValue
        expiration  = json["expiration"].stringValue
    }
}